config/config.go

```diff
@@ -42,6 +42,23 @@ const (
 	SecretFileCustomDashboardsPrometheusPassword = "customdashboards-prometheus-password"
 	SecretFileCustomDashboardsPrometheusToken    = "customdashboards-prometheus-token"
 
+	// External services certificate files
+	SecretFilePrometheusCA                   = "prometheus-ca"
```

Comment:
There's already a number of different ways to add CAs. Does this add another? Instead of a separate CA for each connection, can they all be bundled together in the `additional-ca-bundle`? https://github.com/kiali/kiali/blob/a2f665f52485bf76380ff239b9f7f882377153c1/config/config.go#L104

Reply:
I deprecated all the individual ca_file settings in all the external services (they are now ignored in the new code; log message will be output if one is specified to let people know). There is now one and only one way to add additional CAs - that is via the cabundle ConfigMap. It also now supports auto-rotation. Molecule testing exercises this new functionality.